hrui/js/src/hrui.js

```diff
@@ -189,6 +189,7 @@
   function manipulateDOMOfInlineCustomData () {
     if ($('.Inline_Custom_Data').length) {
```

Comment:
this check was necessary because otherwise `repositionInlineCustomDataBlockInPersonalDetailsTab` wouldn't have anything to move, but why is it necessary to do that check for the new function? What if `. Inline_Custom_Data` is not present? Shouldn't we move the "Extended Demographic" block anyway?

Reply:
You are correct. I initially thought `Inline_Custom_Data` was a determinant to the page rendering before reviewing other section of the code.